man/daemon.xml

```diff
@@ -166,12 +166,22 @@
       following:</para>
 
       <orderedlist>
-        <listitem><para>If <constant>SIGTERM</constant> is received,
-        shut down the daemon and exit cleanly.</para></listitem>
+        <listitem><para>If applicable, the daemon should notify the init system about startup completion or
```

Comment:
init system → service manager

Reply:
(simply since user services should do that too, but they are not started by the "init system" at all, but by the --user service manager)

Reply:
True. I added a separate commit for this change, because the phrase appeared in many places and I think the same change needs to be applied everywhere.

Reply:
The last commit is trivial, even if rather verbose, so I'll upgrade the label.